### PR TITLE
Integrate JPA 3.0 API and Container Feature

### PR DESCRIPTION
**Pull request by jgrassel**
Integrate JPA 3.0 API and Container Feature

Delivers Release Candidate 1 of the JPA 3.0 API.

### CODE REVIEW

**Comment by jgrassel** [comment]
#build

File: dev/com.ibm.ws.jpa.container.core/src/com/ibm/ws/jpa/pxml30/Persistence.java [hunk 1]
```diff
@@ -0,0 +1,736 @@
+/*******************************************************************************
+ * Copyright (c) 2017 IBM Corporation and others.
```
Comment (tkburroughs) [inline dev/com.ibm.ws.jpa.container.core/src/com/ibm/ws/jpa/pxml30/Persistence.java:2]: 2020

Comment (jgrassel) [reply dev/com.ibm.ws.jpa.container.core/src/com/ibm/ws/jpa/pxml30/Persistence.java]: oops thanks for catching that! Fixed!

**Comment by jgrassel** [comment]
#build

**Comment by jhanders34** [comment]
#run-libby-bot

**Comment by jgrassel** [comment]
#build

**Comment by jgrassel** [comment]
#run-libby-bot

**Comment by jgrassel** [comment]
#build

**Comment by jgrassel** [comment]
#run-libby-bot

**Comment by jgrassel** [comment]
#run-libby-bot
#build

**Comment by andrea-pichler** [comment]
No message review requested from L2

**Comment by jgrassel** [comment]
Failed tests are not related to this code update.

**Pull request merged**